### PR TITLE
[AAP-11699] Add update_password command

### PR DESCRIPTION
**Pull request by Dostonbek1**
- Adds a new django-admin command `update_password` to update the given user's password

Solves: [AAP-11699](https://issues.redhat.com/browse/AAP-11699)

### CODE REVIEW

**Review by jamesmarshall24 — DISMISSED** [review]

**Review by mkanoor — COMMENTED** [review]
@Dostonbek1 can we add tests

File: src/aap_eda/core/management/commands/update_password.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        parser.add_argument("-p", "--password", required=True)
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        username = options["username"]
+        password = options["password"]
+        try:
+            user = models.User.objects.get(username=username)
+            user.set_password(password)
+            user.save()
```
Comment (Alex-Izquierdo) [inline src/aap_eda/core/management/commands/update_password.py:34]: I think we should print a message indicating the success of the operation.

File: src/aap_eda/core/management/commands/update_password.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    help = "Update user password."
+
+    def add_arguments(self, parser):
+        parser.add_argument("-u", "--username", required=True)
+        parser.add_argument("-p", "--password", required=True)
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        username = options["username"]
+        password = options["password"]
```
Comment (Alex-Izquierdo) [inline src/aap_eda/core/management/commands/update_password.py:30]: We should check if required args were provided and raise a command error if not.

File: src/aap_eda/core/management/commands/update_password.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        try:
+            user = models.User.objects.get(username=username)
```
Comment (Alex-Izquierdo) [inline src/aap_eda/core/management/commands/update_password.py:32]: I think user model has a check_password method that we could call before.

**Review by Alex-Izquierdo — CHANGES_REQUESTED** [review]
This should have tests

File: src/aap_eda/core/management/commands/update_password.py (hunk 1 above)
Comment (Dostonbek1) [reply src/aap_eda/core/management/commands/update_password.py:34]: We have the message on line 38.

File: src/aap_eda/core/management/commands/update_password.py (hunk 2 above)
Comment (Dostonbek1) [reply src/aap_eda/core/management/commands/update_password.py:30]: `required=True` param in parser takes care of that for us.

File: src/aap_eda/core/management/commands/update_password.py (hunk 1 above)
Comment (Alex-Izquierdo) [reply src/aap_eda/core/management/commands/update_password.py:34]: Right, I commented before read the end. :)

File: src/aap_eda/core/management/commands/update_password.py (hunk 3 above)
Comment (Dostonbek1) [reply src/aap_eda/core/management/commands/update_password.py:32]: Thanks. Working on adding the tests now.

File: src/aap_eda/core/management/commands/update_password.py (hunk 2 above)
Comment (Alex-Izquierdo) [reply src/aap_eda/core/management/commands/update_password.py:30]: good point.

**Review by Alex-Izquierdo — APPROVED** [review]

**Review by mkanoor — APPROVED** [review]

**Pull request merged**